src/Mutator/Arithmetic/Assignment.php

```diff
@@ -74,7 +76,11 @@ public static function getDefinition(): Definition
      */
     public function mutate(Node $node): iterable
     {
-        yield new Node\Expr\Assign($node->var, $node->expr, $node->getAttributes());
+        yield new Node\Expr\Assign(
+            $node->var,
+            $node->expr,
+            array_diff_key($node->getAttributes(), array_flip(['origNode'])),
```

Comment:
Given how it is used everywhere, maybe we can use a utility there? `NodeAttributes::getAllExceptOriginalNode()`?

Reply:
that's what I've already planned here: https://github.com/infection/infection/pull/2448#discussion_r2430331567. Yes, I will create a static class/method for it